src/IssueViz/Editor/SelectedIssueTagging/Highlighting/IssueHighlightViewTaggerProvider.cs

```diff
@@ -0,0 +1,71 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Tagging;
+using Microsoft.VisualStudio.Utilities;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Editor.SelectedIssueTagging.Highlight
+{
+    [Export(typeof(IViewTaggerProvider))]
+    [ContentType("code")]
+    [TextViewRole(PredefinedTextViewRoles.PrimaryDocument)]
```

Comment:
In TaggerProvider we used TextViewRole `Document` instead of `PrimaryDocument`. I see that primary document is `Applies to text views of documents that are open for mainline editing, excluding auxiliary views of documents.` -- I wonder what they mean by auxiliary views; peek definition for example? Shame that the documentation is so vague.

Reply:
Yep, not sure. We can try different combinations and see if we can limit it to "real" editable documents. 
I think comparison views are another type of auxiliary document.